rust/decentraland-godot-lib/src/scene_runner/scene_manager.rs

```diff
@@ -194,6 +194,17 @@ impl SceneManager {
         GodotString::default()
     }
 
+    #[func]
+    fn get_scene_id_by_parcel_position(&self, parcel_position: Vector2i) -> i32 {
+        for scene in self.scenes.values() {
+            if scene.definition.parcels.contains(&parcel_position) {
```

Comment:
Skip global scenes before this `if let SceneType::Global(_) = scene.scene_type { continue; }